test/goldens/end2end/angular_components-0.10.0.json_report.md

```diff
@@ -0,0 +1,110 @@
+## 5/20 Packaging conventions
+
+### [x] 0/10 points: Provide a valid pubspec.yaml
```

Comment:
Nit: maybe add an extra newline after the headers? I think `package:markdown` does not need it, but it feels like it is missing from here.

Reply:
Done